pkg/group_snapshotter/group_snapshotter_test.go

```
@@ -0,0 +1,343 @@
+			ReadyToUse:      true,
+		},
+	}, nil
+}
+
+func (f *fakeCSIServer) GroupControllerGetCapabilities(ctx context.Context, req *csi.GroupControllerGetCapabilitiesRequest) (*csi.GroupControllerGetCapabilitiesResponse, error) {
+	return &csi.GroupControllerGetCapabilitiesResponse{}, nil
+}
+
+func startFakeCSI(t *testing.T, fake *fakeCSIServer) (conn *grpc.ClientConn, cleanup func()) {
+	t.Helper()
+	lis, err := net.Listen("tcp", "127.0.0.1:0")
```

golangci-lint (2.12.2)

[error] net.Listen must not be called. use (*net.ListenConfig).Listen

(noctx)